addons/payment_adyen/models/payment_transaction.py

```diff
@@ -218,13 +320,20 @@ def _process_feedback_data(self, data):
         if self.provider != 'adyen':
             return
 
+        # Extract or assume the event code. If none is provided, the feedback data originate from a
+        # direct payment request whose feedback data shares the same payload as an 'AUTHORIZATION'
```

Comment:
AUTHORIZATION or AUTHORISATION ? :D

Reply:
Both are correct! =D
Adyen sadly use "authorisation"... It hurts me a lot. Ok, ok, I'll track down any "authorization" left behind... xD

Reply:
`resolved`

Reply:
:thinking:

Reply:
:innocent: